compiler/compiler.go

```diff
@@ -187,6 +187,11 @@ func CompileWithCommand(cmd *exec.Cmd, workingDir string) (problems types.BuildP
 					problem.File = filepath.Join(workingDir, groups[1])
 				}
 
+				if string(filepath.Separator) != "\\" {
+					problem.File = strings.Replace(problem.File, "\\", "/", -1)
```

Comment:
Why replace `\` to `/`?  Shouldn't you replace the non-os version to the OS version?  That code is fine on Linux, but most SA:MP code is written on Windows, which should use `\`.

Reply:
The function [filepath.Clean](https://golang.org/pkg/path/filepath/#Clean) accept both for windows, but only `/` for linux

Reply:
But thinking now, the condition is no needed, can just replace because windows will accept both

Reply:
Yeah the replacement is not necessary, Windows apps work fine with `/` and the compiler is no exception. The clean is all that's needed here.

Reply:
Is necessary on linux

Reply:
Oh yeah I misread it as the opposite way around!